tutorials/nnvm/deploy_ssd.py

```diff
@@ -32,17 +33,20 @@
 #   echo "set(USE_SORT ON)" > config.mk
 #   make -j8
 #
-# .. note::
-#
-#   Currently we support compiling SSD on CPU only.
-#   GPU support is in progress.
-#
 
 model_name = "ssd_resnet50_512"
 model_file = "%s.zip" % model_name
 test_image = "dog.jpg"
 dshape = (1, 3, 512, 512)
 dtype = "float32"
+
+# Target settings
+# Use these commented settings to build for cuda.
+#target = 'cuda'
+#ctx = tvm.gpu(0)
+# Use these commented settings to build for opencl.
+#target = 'opencl'
+#ctx = tvm.gpu(0)
```

Comment:
if I remember correctly, for opencl it should be tvm.opencl(0) or tvm.cl(0), isn't it?

Reply:
Yes, sry I forgot to change.